modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -117,6 +122,38 @@ Undocumented props are spread to the underlying `<button>` element.
 
 <ArgsTable of={DeleteButton} />
 
+### Hyperlink
+
+Hyperlinks should be used when you want to navigate away from the current page or to an anchor on
+the current page.
+
+<ExampleCodeBlock code={Link} />
+
+Hyperlinks also have an `inverse` variant. Use this variant when you need to place a Hyperlink on a
+dark or colorful background such as `blueberry400`.
+
+<ExampleCodeBlock code={LinkInverse} />
+
+#### Props
+
+Undocumented props are spread to the underlying `<a>` element.
+
+<ArgsTable of={Hyperlink} />
+
+### HyperlinkExternal
+
+If your link should open in a new tab or window use the `<HyperlinkExternal>` component. It will let
+users know about the behavior by adding an icon, `extLinkIcon`. You can override `aria-label` of the
+icon by using the `iconLabel` attribute.
```

Comment:
We've been moving away from default translations. In this case, it might make sense. I think we should add a bit about translation though, describing the purpose of `iconLabel`. A callout will help with translation efforts. Canvas Kit doesn't take care of translations itself as many teams using Canvas Kit have different translation solutions.